crates/router/tests/connectors/tsys.rs

```diff
@@ -286,9 +296,14 @@ async fn should_refund_succeeded_payment_multiple_times() {
 #[actix_web::test]
 async fn should_sync_refund() {
     let refund_response = CONNECTOR
-        .make_payment_and_refund(payment_method_details(), None, get_default_payment_info())
+        .make_payment_and_refund(
+            payment_method_details(100),
+            None,
+            get_default_payment_info(),
+        )
         .await
         .unwrap();
+    tokio::time::sleep(Duration::from_secs(10)).await;
```

Comment:
Do we still need this sleep time?

Reply:
Yes in this transaction is registered after a lag, that is why sleep is added